homeassistant/components/overkiz/strings.json

```diff
@@ -14,6 +14,7 @@
     },
     "error": {
       "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "cozytouch_unsupported_hardware": "Your CozyTouch hardware is not manufactured by Overkiz, and thus not supported by this integration.",
```

Comment:
The word "manufactured" seems odd to me.
I think the sentence could be clearer with :
`Your CozyTouch hardware is not supported by Overkiz.`

Reply:
Please use description_placeholders so the unsupported brand name can be set from code and the error be generic.

Reply:
Thanks, both good suggestions. Fixed in 8c59db6fc6d3fc2bb4e76175e639db255f79da06.

This even helped me get rid of a similar string, that was a bit vague and internal.